server/lib/chat/chat-service.ts

```diff
@@ -180,7 +186,7 @@ export default class ChatService {
     }
 
     const users = await getUsersForChannel(originalChannelName)
-    return users.map(u => u.userName)
+    return users.map<ChatUser>(u => u.userName)
```

Comment:
nit: this generic param is unnecessary, it can be easily inferred. If you'd prefer to note that this is a `ChatUser` and not a `string` (which maybe you can see why I don't find this type to provide much value :) ), do so by notating the return type on the function (which you should generally always be doing for non-simple methods, and this method certainly isn't simple)